modules/ob-installing-buildkit-strategy.adoc

```diff
@@ -0,0 +1,22 @@
+// This module is included in the following assembly:
+//
+// builds/installing-sample-build-strategies.adoc
+
+:_content-type: PROCEDURE
+[id="installing-buildkit-strategy_{context}"]
+= Installing buildkit strategy
+
+You can install the `buildkit` strategy at cluster level. By installing the `buildkit` strategy at cluster level, you can share it across different namespaces within your cluster.
```

Comment:
Same comment as above: " You can share the `buildkit` strategy across different namespaces within your cluster by installing it at the  cluster level."